src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IUserReleaseRoleRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
+{
+    public interface IUserReleaseRoleRepository
+    {
+        public Task<UserReleaseRole> Create(Guid userId,
+            Guid releaseId,
+            ReleaseRole role);
```

Comment:
Good spot, as part of this work we don't change the UserReleaseRole model, which doesn't yet have auditing fields `Created` and `CreatedById`. It would be a nice to have though to see who is assigning release permissions, and when, so I'll create a ticket for it -  EES-2326